CODEOWNERS

```diff
@@ -5,4 +5,4 @@
 # For more details, read the following article on GitHub: https://help.github.com/articles/about-codeowners/.
 
 # These are the default owners for the whole content of this repository. The default owners are automatically added as reviewers when you open a pull request, unless different owners are specified in the file.
-* @shilpa-padgaonkar @rartych @patrice-conil
+* @shilpa-padgaonkar @rartych @patrice-conil @RubenBG7 @jordonezlucena
```

Comment:
```suggestion
* @shilpa-padgaonkar @rartych @patrice-conil @RubenBG7 @jordonezlucena @PedroDiez
```

Reply:
Have you followed the instructions:
_To subscribe / unsubscribe to the mailing list of this Sub Project and thus be / resign as Contributor please visit https://lists.camaraproject.org/g/sp-com._ ?